DatadogLogs/Sources/Feature/MessageReceivers.swift

```diff
@@ -61,101 +61,6 @@ internal struct LogMessageReceiver: FeatureMessageReceiver {
     }
 }
 
-/// Receiver to consume a Crash Log message as Log.
-internal struct CrashLogReceiver: FeatureMessageReceiver {
-    /// Time provider.
-    let dateProvider: DateProvider
-    let logEventMapper: LogEventMapper?
-
-    /// Process messages receives from the bus.
-    ///
-    /// - Parameters:
-    ///   - message: The Feature message
-    ///   - core: The core from which the message is transmitted.
-    func receive(message: FeatureMessage, from core: DatadogCoreProtocol) -> Bool {
-        guard case let .payload(crash as Crash) = message else {
-            return false
-        }
-
-        return send(report: crash.report, with: crash.context, to: core)
-    }
-
-    private func send(report: DDCrashReport, with crashContext: CrashContext, to core: DatadogCoreProtocol) -> Bool {
-        // The `report.crashDate` uses system `Date` collected at the moment of crash, so we need to adjust it
-        // to the server time before processing. Following use of the current correction is not ideal, but this is the best
-        // approximation we can get.
-        let date = (report.date ?? dateProvider.now)
-            .addingTimeInterval(crashContext.serverTimeOffset)
-
-        var errorAttributes: [AttributeKey: AttributeValue] = [:]
-
-        // Set crash attributes for the error
-        errorAttributes[DDError.threads] = report.threads
-        errorAttributes[DDError.binaryImages] = report.binaryImages
-        errorAttributes[DDError.meta] = report.meta
-        errorAttributes[DDError.wasTruncated] = report.wasTruncated
```

Comment:
suggestion/ We can remove these `DDError` constants. They were initially used when RUM and Logs relied on error attributes for crash-specific fields. Since we already moved to the type-safe `DDCrashReport` in RUM and now we remove this concept from Logs, they're no longer needed.